build.gradle

```diff
@@ -42,6 +42,10 @@ dependencies {
 
   // Sentry reporting
   compile "io.sentry:sentry-spring:1.7.30"
+
+  // test
+  testImplementation('org.junit.jupiter:junit-jupiter-engine:5.2.0')
```

Comment:
These should not be needed, the test starter is included at line 30 already.

Reply:
Removed, cheers.